resources/public/api/conf/2.0/application.yaml

```diff
@@ -0,0 +1,10973 @@
+          "withdrawalAmount": 1000.00,
+          "withdrawalChargeAmount": 250.00,
+          "withdrawalChargeAmountYTD": 500.00,
+          "fundsDeductedDuringWithdrawal": true,
+          "withdrawalReason": "Regular withdrawal",
+          "automaticRecoveryAmount": 250.00
+        }
+        </pre>
+                    </td>
+                    <td>
+                        <p>HTTP status: <code class="code--slim">404 (Not Found)</code></p>
+        <pre class="code--block">
+        {
+          "code": "INVESTOR_ACCOUNTID_NOT_FOUND",
+          "message": "Enter a real accountId"
+        }
+        </pre>
+                    </td>
+                </tr>
+                <tr>
+                    <td>
+                        <p>The accept header is invalid</p>
+                        <p class="code--block">
+                            <strong>lisaManagerReferenceNumber:</strong><br>
+                            <a href="#section/Testing">Use your test user profile</a><br>
+                            <br>
+                            <strong>accountId:</strong><br>1234567890
+                            <br>
+                            <br>
+                            <strong>Accept:</strong><br> application/vnd.hmrc.1.0
+                        </p>
+                    </td>
+                    <td>
+        <pre class="code--block">
+        {
+          "claimPeriodStartDate": "2017-12-06",
+          "claimPeriodEndDate": "2018-01-05",
+          "withdrawalAmount": 1000.00,
+          "withdrawalChargeAmount": 250.00,
+          "withdrawalChargeAmountYTD": 500.00,
+          "fundsDeductedDuringWithdrawal": true,
+          "withdrawalReason": "Regular withdrawal",
+          "automaticRecoveryAmount": 250.00
+        }
+        </pre>
+                    </td>
+                    <td>
+                        <p>HTTP status: <code class="code--slim">404 (Not Found)</code></p>
+        <pre class="code--block">
+        {
+          "code": "MATCHING_RESOURCE_NOT_FOUND",
+          "message": "A resource with the name in the request can not be found in the API"
+        }
+        </pre>
+                    </td>
+                </tr>
+                <tr>
+                    <td>
+                        <p>Withdrawal charge has already been reported</p>
+                        <p class="code--block">
+                            <strong>lisaManagerReferenceNumber:</strong><br>
+                            <a href="#section/Testing">Use your test user profile</a><br>
+                            <br>
+                            <strong>accountId:</strong><br>0000000409
+                        </p>
+                    </td>
+                    <td>
+        <pre class="code--block">
+        {
+          "claimPeriodStartDate": "2017-12-06",
+          "claimPeriodEndDate": "2018-01-05",
+          "withdrawalAmount": 1000.00,
+          "withdrawalChargeAmount": 250.00,
+          "withdrawalChargeAmountYTD": 
```

Comment:
This field is missing a description. Please add, thanks.

        marketValueStocksAndShares:
          description: The total value of the stocks and shares LISA account. Give the value to the nearest whole pound. Do not include decimal places. For example, send 54.56 as 55. If you give a value for marketValueStocksAndShares, give a value of 0 for marketValueCash and annualSubsCash.
          maximum: 999999
          minimum: 0
          type: integer
          format: int32
          example: 1000